drivers/sdhc/rcar_mmc.c

```diff
@@ -376,7 +376,7 @@ static int rcar_mmc_reset(const struct device *dev)
 
 #ifdef CONFIG_RCAR_MMC_SCC_SUPPORT
 		/* tune if this reset isn't invoked during tuning */
-		if (can_retune && (ios.timing == SDHC_TIMING_HS200)) {
+		if (can_retune) {
```

Comment:
Will this cause retuning in all modes? I am asking mostly about modes that does not require tuning.

Reply:
So, it can cause issues in the future. Could you please add TODO or FIXME regarding this?